app/src/androidTest/java/com/github/fribourgsdp/radio/activities/WorkingJoinGameActivityTest.kt

```diff
@@ -32,15 +35,16 @@ import org.junit.runner.RunWith
  */
 @RunWith(AndroidJUnit4::class)
 class WorkingJoinGameActivityTest {
-
-    @get:Rule
-    var mRuntimePermissionRule = GrantPermissionRule.grant(Manifest.permission.CAMERA)
     @get:Rule
-    var mRuntimePermissionRule2 = GrantPermissionRule.grant(Manifest.permission.VIBRATE)
+    var joinActivityRule = ActivityScenarioRule(WorkingJoinGameActivity::class.java)
 
+    private val ctx: Context = ApplicationProvider.getApplicationContext()
 
     @Before
     fun initIntent() {
+        // TODO: Remove this when mocked
```

Comment:
You must not forget to mock this.

Reply:
I don't know if I'll personally do it but we will attribute it to someone in the next sprint 👍🏻